### ISSUE TITLE
Release tag does not match version name

### ISSUE THREAD

**Issue by jxareas** [BrunoMazzocchi/crunchyanime#1]
As implied, the current [alpha release](https://github.com/BrunoMazzocchi/crunchyanime/releases/tag/v0.1.0-alpha) does not match the version name.

You probably forgot to update the version in the `pubspec.yaml` before releasing, as it is identified as: https://github.com/BrunoMazzocchi/crunchyanime/blob/f2b4594ff2aba0a4e72650c61119f4641ab5cd1b/pubspec.yaml#L20

which is the one used in the `build.gradle` file from the `:app` module. 
https://github.com/BrunoMazzocchi/crunchyanime/blob/f2b4594ff2aba0a4e72650c61119f4641ab5cd1b/android/app/build.gradle#L53

**Comment by BrunoMazzocchi** [comment]
You're right. I completely forgot to update the version. Once I update the BLoC implementation, I'll post it correctly. Thanks in advance.
